integration-tests/NAME.IntegrationTests/Elasticsearch/ElasticsearchVersionResolverTests.cs

```diff
@@ -0,0 +1,37 @@
+using NAME.ConnectionStrings;
+using NAME.Core;
+using NAME.Elasticsearch;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace NAME.IntegrationTests.Elasticsearch
+{
+    public class ElasticsearchVersionResolverTests
+    {
+        [Fact]
+        [Trait("TestCategory", "Integration")]
+        public async Task GetVersions_SpecificVersion()
+        {
+            ElasticsearchVersionResolver resolver = new ElasticsearchVersionResolver(new StaticConnectionStringProvider($"http://{ Constants.SpecificMongoHostname }:9200"), 10000, 10000);
```

Comment:
Typo, should be `SpecificElasticsearchHostname` instead of `SpecificMongoHostname`